README.md

```diff
@@ -144,6 +144,7 @@ include_capi_no_bridge
 * `include_zipkin`: Flag to include tests for Zipkin tracing. `include_routing` must also be set for tests to run. CF must be deployed with `router.tracing.enable_zipkin` set for tests to pass.
 * `use_http`: Set to true if you would like CF Acceptance Tests to use HTTP when making api and application requests. (default is HTTPS)
 * `use_log_cache`: Set to false if you don't want CF Acceptance Tests to use Log Cache for reading application logs. (default is true)
+* `use_kubernetes_istio_proxy`: Set to true if running against a "Cloud Foundry for Kubernetes" environment using [cf-k8s-networking](https://github.com/cloudfoundry/cf-k8s-networking) with an Istio Proxy for ingress routing. (default is `false`)
```

Comment:
Open to suggestions for this config flag name. We left it a bit generic because we may need to tweak tests around other Gorouter-specific behavior in the future.

Reply:
Would it make sense to remove the Kubernetes reference in these? What would we call it if we'd enabled using Istio in cf-deployment?

Reply:
Probably `use_istio_ingress` or `istio_ingress_enabled` or something. 🤔  I agree we could get rid of the `kubernetes` part of the name and that the "istio" part is more important. I imagine if we changed ingress implementations that this behavior would change as well.

Reply:
`use_istio_ingress` sounds good to me 👍